internal/providers/terraform/azure/automation_dsc_nodeconfiguration.go

```diff
@@ -1,22 +1,21 @@
 package azure
 
 import (
+	"github.com/infracost/infracost/internal/resources/azure"
 	"github.com/infracost/infracost/internal/schema"
 )
 
-func GetAzureRMAutomationDscNodeconfigurationRegistryItem() *schema.RegistryItem {
+func getAzureRMAutomationDscNodeconfigurationRegistryItem() *schema.RegistryItem {
```

Comment:
```suggestion
func getAutomationDSCNodeConfigurationRegistryItem() *schema.RegistryItem {
```
Sorry, I missed this one